apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.TargetedSessionPool.cs

```diff
@@ -651,7 +651,7 @@ internal DatabaseStatistics GetStatisticsSnapshot()
             /// </remarks>
             /// <param name="cancellationToken">An optional token for canceling the call.</param>
             /// <returns>A task which will complete when the session pool has reached its minimum size.</returns>
-            internal async Task WaitForPoolAsync(CancellationToken cancellationToken = default)
+            internal async Task WhenPoolReady(CancellationToken cancellationToken = default)
```

Comment:
nit: Maybe change the summary here as well.

Reply:
Yeah, I was lazy about that one as it's internal ;) Will fix - thanks for keeping me honest.